backend/courses/util.py

```diff
@@ -34,22 +34,28 @@ def separate_course_code(course_code):
     raise ValueError(f"Course code could not be parsed: {course_code}")
 
 
-def get_or_create_course(dept_code, course_id, semester):
+def get_or_create_course(dept_code, course_id, semester, report_created=False):
```

Comment:
I'm not really a fan of `report_created` I think we should stick to the django spec and always report it